src/util/seeds.ts

```diff
@@ -280,15 +280,23 @@ export class ScopedSeed {
       }
     }
 
+    if (this.isExcluded(url)) {
+      return false;
+    }
+
+    return { url, isOOS };
+  }
+
+  isExcluded(url: string) {
     // check exclusions
     for (const e of this.exclude) {
       if (e.test(url)) {
         //console.log(`Skipping ${url} excluded by ${e}`);
-        return false;
+        return true;
```

Comment:
Good catch!!

Reply:
heh, well the logic was flipped here as it was part of isIncluded() but now part of the separate isExcluded() method.

Reply:
Ah true, don't mind me 😂